### ISSUE TITLE
Extraction from jade templates, mixins don't work

### ISSUE THREAD

**Issue by gerotakke** [zaach/jsxgettext#99]
As the title says, extraction doesn't work when the translated string is a parameter of a mixin.
I've hacked around a bit and found that there's a token type "call" that isn't handled in parsers/jade.js.
I got it to work by simply adding this:
```javascript
      case 'call':
        append.call(token, extractGettext(token.args));
      break;
```
My tests look very promising but it seems a bit too straightforward.
Should I submit a PR? Anything else I can do?

**Comment by BYK** [comment]
I'd just submit a PR and make sure it has new tests to verify the behavior (and also not break any old tests).

Until me or someone else has time to implement proper parsing/AST based extraction for languages other than JavaScript, I don't think we can do much better.

**Comment by gerotakke** [comment]
I just noticed that strings inside of mixins aren't extracted as well, also after the changes in PR. Will look into it as soon as I have the time.

**Comment by BYK** [comment]
Thank you!

**Comment by BYK** [comment]
Guessing this is closed by #100?

**Comment by gerotakke** [comment]
Yep, thanks!
